com.unity.ml-agents/Runtime/Actuators/BaseActuator.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using Unity.MLAgents.Policies;
+
+namespace Unity.MLAgents.Actuators
+{
+    public abstract class BaseActuator : IActuator
```

Comment:
This class also managed the updating of its own action buffer from the entire 'behavior' buffer of actions